### PR TITLE
Add a cask for Kancolle

### PR DESCRIPTION
**Pull request by sachin21**
# WHAT

Added a cask for the Kancolle(艦隊これくしょん-艦これ-)
Kancolle is a online pc game. and kanmemo is a browser for the Kancolle.
# REF
- https://github.com/ioriayane/KanmusuMemory
- http://www.dmm.com/netgame/feature/kancolle.html (Japanese official site)

### CODE REVIEW

File: Casks/kanmemo.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask :v1 => 'kanmemo' do
+  version '0.15'
+  sha256 'af64ae0846ab0b4366693bc602a81ba7e626bafee820862594c4bcbf92acfcef'
+
+  url "http://relog.xii.jp/download/kancolle/KanmusuMemory-0.15-mac.dmg"
```
Comment (fanquake) [inline Casks/kanmemo.rb]: use #{version} here

Comment (sachin21) [inline Casks/kanmemo.rb]: Thanks, I fixed it

File: Casks/kanmemo.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+cask :v1 => 'kanmemo' do
```
Comment (vitorgalvao) [inline Casks/kanmemo.rb]: [Token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_token_reference.md).

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in b83f8ecf3b80b43feb6bd4a670e2e5938cb1e898. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by sachin21** [comment]
Sorry. I'm beginner.
Thank you very much.

**Pull request closed without merging**